src/thinblock.cpp

```diff
@@ -891,6 +891,12 @@ string CThinBlockData::MempoolLimiterBytesSavedToString()
     return ss.str();
 }
 
+// Preferential Thinblock Timer:
```

Comment:
The comment is good, but our developer-notes.md states:

> [...] use doxygen-compatible comment blocks for functions, methods and fields.

I believe even if just used for the general description, this is better practice.
It makes method header comments stand out much more from regular comments in a file, and much of the prose (like preconditions) can be made good use of in generated documentation.